ispyb/sp/mxacquisition.py

```diff
@@ -68,6 +68,17 @@ def __init__(self):
   _fluo_mapping_roi_params =\
     StrictOrderedDict([('id',None), ('start_energy',None), ('end_energy',None), ('element',None), ('edge',None), ('r',None), ('g',None), ('b',None)])
 
+  _xray_centring_params =\
+   StrictOrderedDict([('id',None), ('grid_info_id',None), ('method',None), ('status',None), ('x',None), ('y',None)])
+
+  @classmethod
+  def get_xray_centring_result_params(cls):
+      return copy.deepcopy(cls._xray_centring_params)
+
+  def upsert_xray_centring_result(self, values):
```

Comment:
Are you happy for me to change this to a keyword-argument format similar to [```upsert_program_ex```](https://github.com/DiamondLightSource/ispyb-api/blob/e8114b303753da7e0aecb53fec1cedc916a8fe64/ispyb/sp/mxprocessing.py#L126-L129)?

Reply:
Absolutely - that sounds good.